letsencrypt/cli.py

```diff
@@ -669,7 +682,12 @@ def determine_verb(self):
 
         for i, token in enumerate(self.args):
             if token in self.VERBS:
-                self.verb = token
+                verb = token
+                if verb == "certonly":
+                    verb = "auth"
```

Comment:
I personally don't have a strong preference on aliases. (I secretly want to make a verb "allthethings" so we can say `letsencrypt allthethings`).

With that said, how much extra work would it be @pde to flip this logic here and have "certonly" be the main version of the verb? I could be wrong, but I don't think it'd be that hard and it'd get rid of awkward behavior like:
```
$ letsencrypt --help certonly       

usage: 
  letsencrypt [SUBCOMMAND] [options] [-d domain] [-d domain] ...

The Let's Encrypt agent can obtain and install HTTPS/TLS/SSL certificates.  By
default, it will attempt to use a webserver both for obtaining and installing
the cert. Major SUBCOMMANDS are:

  (default) run        Obtain & install a cert in your current webserver
  certonly             Obtain cert, but do not install it (aka "auth")
  install              Install a previously obtained cert in a server
  revoke               Revoke a previously obtained certificate
  rollback             Rollback server configuration changes made during install
  config_changes       Show changes made to server config during installation

optional arguments:
  -h, --help            show this help message and exit
  -c CONFIG_FILE, --config CONFIG_FILE
                        config file path (default: None)

auth:
  Options for modifying how a cert is obtained

  --csr CSR             Path to a Certificate Signing Request (CSR) in DER
                        format; note that the .csr file *must* contain a
                        Subject Alternative Name field for each domain you
                        want certified. (default: None)
```

Where's my help on `certonly`? :(